tex/latex/biblatex/biblatex.def

```diff
@@ -2803,6 +2804,26 @@
 
 \newbibmacro*{date}{\printdate}
 
+\DeclareFieldFormat{legaldates}{\mkbibparens{#1}}
+\newbibmacro*{legaldates}{
+  \ifboolexpr{%
+      test {\iffieldundef{filedyear}} and%
+      test {\iffieldundef{grantedyear}} and%
+      test {\iffieldundef{expiredyear}}%
+    }%
+  {}%
+  {%
+      \newunit%
+      \printtext[legaldates]{%
+        \iffieldundef{filedyear}{}{\bibstring{filed}\space \printfileddate}%
```

Comment:
I haven't tested it, but something like
```suggestion
        \printfileddate
```
would be enough here if we also define
```latex
\DeclareFieldFormat{fileddate}{\bibstring{filed}\space#1}
```

Reply:
work done